main_app/models.py

```diff
@@ -36,3 +36,9 @@ class create_host_model(models.Model):
     def __str__(self):
         return self.vm_name
 
+class operating_system_model(models.Model):
+    os_name = models.CharField(max_length=15)
+    os_location = models.CharField(max_length=100)
```

Comment:
Here you need to add another parameter that is choices=[list of all operating systems]

Reply:
@yadneshk 
I think we need to add it in create_host_model class in model.py file

Reply:
Only make sure values don't get repeated